src/dynamic_boundary_conditions/hirds_rainfall_data_to_db.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 """
-@Script name: hirds_depth_data_to_db.py
+@Script name: hirds_rainfall_data_to_db.py
 @Description: Store rainfall data of all the sites within the catchment area in the database.
 @Author: pkh35
 @Date: 20/01/2022
 @Last modified by: sli229
-@Last modified date: 27/09/2022
+@Last modified date: 28/09/2022
```

Comment:
Does this get updated automatically or do you need to do it?

Reply:
manual update.